claim.md

```diff
@@ -1,85 +1,89 @@
 # 💬 Claim
 
-In this section, you'll play the role of a <span class="label-role claimer">claimer</span>:  
-* You'll make a claim about yourself;  
-* A claim in itself has no value; you'll have to request an attestation of your claim. For a claim to become valid in the eyes of <span class="label-role verifier">verifiers</span>, it needs to be attested by a trusted entity.
+In this section, you'll play the role of a <span class="label-role claimer">claimer</span>.
 
-Then an <span class="label-role attester">attester</span> will pick up your request and hopefully attest your claim. We'll look into this in the next steps - for now, let's just focus on our claim.    
+* You'll first make a claim about yourself in the form of a `Claim` object;
+* But a claim in itself has no value. To become valid in the eyes of <span class="label-role verifier">verifiers</span>, it needs to be attested by an entity that <span class="label-role verifier">verifiers</span> trust: an <span class="label-role attester">attester</span>. So you'll create a `RequestForAttestation` object from your `Claim` object, so that an <span class="label-role attester">attester</span> can attest it.
 
+We'll look into the attestation in the next steps - for now, let's just focus on your claim.
 
-> 💡 KILT is permissionless.   
-> Anyone/anything can make a claim about themselves.
+> 💡 KILT is an open system.
+> Anyone/anything can make a claim about themselves. But a claim only has value if the verifier trusts the attester.
 
-<!-- and a *RequestForAttestaion* object, which we will share with the other participants, so that they can attest it. -->
-  
-## Get your identity as a claimer  
-In the previous Identity step in this tutorial, you've generated two identities.    
-You'll need the first mnemonic you've created; it's referred to as `<CLAIMER'S MNEMONIC>` in the code snippet below.   
+## Create a file
 
-## Code: create file 
-
-Create a new file `2-claim.js`. 
-All the following code needs to go into this file.  
+Create a new file `2-claim.js`.
+All of the code for this step needs to go into this file.
 
 ## Code: create a claim
-We'll create a claim using the provided ctype and the claimer identity.  
-Paste the following in `2-claim.js` (make sure to replace the `<CLAIMER'S MNEMONIC>`).  
 
-```javascript 
+In the previous step, you've generated two mnemonics and identities.
+You'll now need the first mnemonic you've created; it's referred to as `<CLAIMER'S MNEMONIC>` in the code snippet below.
+
+We'll create a claim using the provided CTYPE and the claimer identity.  
```

Comment:
Claim and Claimer are not highlighted as with other places. 

e.g. `<span class="label-role claimer">claimer</span>`

Reply:
Fixed, ty